weblate/vcs/base.py

```diff
@@ -506,8 +506,13 @@ def parse_changed_files(self, lines: List[str]) -> Iterator[str]:
         """Parses output with changed files."""
         raise NotImplementedError()
 
-    def list_upstream_changed_files(self):
-        """List files missing upstream."""
+    def list_upstream_changed_files(self, **kwargs):
```

Comment:
Better to include the parameter in the signature:

```suggestion
    def list_changed_files(self, compare_to:Optional[str] = None):
```

Also name of the method should reflect the change.